straxen/plugins/peaklets/peaklet_classification_som.py

```diff
@@ -298,3 +269,17 @@ def compute_wf_attributes(data, sample_length, n_samples: int):
         quantiles[i] = cumsum_steps[1:] - cumsum_steps[:-1]
 
     return quantiles
+
+
+@export
+class PeakletSOMClass(PeakletClassificationSOM):
```

Comment:
This plugin is used in the default straxen context as the provided data-type needs to be different from `peaklet_classification`. The rest is inherited from above.